Projects/Feature/SigninFeature/Sources/Scene/SigninView.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+import BaseFeature
+import DesignSystem
+import GlobalThirdPartyLibrary
+import ViewUtil
+import UtilityModule
+
+struct SigninView: View {
+    @StateObject var container: MVIContainer<SigninIntentProtocol, SigninStateProtocol>
+    var intent: any SigninIntentProtocol { container.intent }
+    var state: any SigninStateProtocol { container.model }
+
+    var body: some View {
+        VStack(spacing: 0) {
+            Text("STUDENT\nMANAGEMENT\nSERVICE")
+                .multilineTextAlignment(.center)
+                .padding(.top, 124)
+                .font(.sms(.headline1))
+                .foregroundColor(.sms(.system(.white)))
+
+            Text("학생 정보 통합관리 서비스")
+                .font(.sms(.title2))
+                .foregroundColor(.sms(.system(.white)))
+                .padding(.top, 16)
+
+            Spacer()
+
+            GAuthButtonView { code in
+                container.intent.signin(code: code)
+            }
+            .padding(.horizontal, 20)
+            .frame(height: 50)
+            .padding(.bottom, 88)
+        }
+        .background {
+            SMSImage(.background)
+        }
+        .ignoresSafeArea()
+        .alert(
```

Comment:
```suggestion
            GAuthButtonView { code in
                container.intent.signin(code: code)
            }
            .padding(.horizontal, 20)
            .frame(height: 50)
            .padding(.bottom, 54)
        }
        .background {
            SMSImage(.background)
                .ignoresSafeArea()
        }
        .alert(
```

safeArea를 전체 View에 주지말고 background에만 주고 GAuthButton의 bottom padding을 54로 해야할거같아요.

Reply:
iPhone se같은거에 당하면 그냥 그대로 88만큼 떨어질거라